controller_manager:
  ros__parameters:
    update_rate: 1000

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    joint_effort_controller_j_0:
      type: forward_command_controller/ForwardCommandController

joint_effort_controller_j_0:
  ros__parameters:
    joints:
      - j_0
    interface_name: effort

/lip/controller_manager:
  ros__parameters:
    update_rate: 1000

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    joint_effort_controller_j_0:
      type: forward_command_controller/ForwardCommandController

/lip/joint_effort_controller_j_0:
  ros__parameters:
    joints:
      - j_0
    interface_name: effort
